proxy-robots-module/src/main/java/orwell/proxy/Cli.java

```diff
@@ -0,0 +1,86 @@
+package orwell.proxy;
+
+import org.apache.commons.cli.*;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import orwell.proxy.config.ConfigCli;
+import orwell.proxy.config.EnumConfigFileType;
+
+import java.io.File;
+
+/**
+ * Created by miludmann on 5/5/15.
+ */
+public class Cli {
+    final static protected String DEFAULT_CONFIG_FILEPATH = "/configuration.xml";
```

Comment:
Is it OK to hard code the path like that ? (I think it might be because you are referring to inside the jar so you would have no issues with different folder separators depending on the platform)